pkgs/data/fonts/klingon-piqad-hasta/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, fetchzip }:
+
+let
+  pname = "klingon-piqad-hasta";
+  version = "203";
+in (fetchzip {
+  name = "${pname}-${version}";
```

nit: you can use `pname`+`version` directly:
```suggestion
(fetchzip {
  pname = "klingon-piqad-hasta";
  version = "203";
```